assets/src/edit-story/elements/text/karma/edit.karma.js

```diff
@@ -197,7 +197,8 @@ describe('TextEdit integration', () => {
           height: heightAfterExitingEditMode,
         } = frame.getBoundingClientRect();
 
-        expect(initialHeight).toEqual(heightAfterExitingEditMode);
+        let difference = Math.abs(heightAfterExitingEditMode - initialHeight);
+        expect(difference).toBeLessThan(1);
```

Comment:
Hmm, why less than 1?

Reply:
Is this because of rounding errors I suggest using `toBeCloseTo` instead.

Reply:
For not yet clear reasons (where the discrepancy occurs) the height can be e.g. 409 vs 409.39, for now, I'm adding this 1px difference tolerance.

Reply:
Hmm, this small difference can actually cause some other issues, for example:

Edit mode:
<img width="305" alt="Screenshot 2020-09-09 at 16 56 45" src="https://user-images.githubusercontent.com/3294597/92608002-201d1280-f2b5-11ea-96ff-bd3d920453e4.png">

Exiting edit mode:
<img width="268" alt="Screenshot 2020-09-09 at 16 56 38" src="https://user-images.githubusercontent.com/3294597/92608010-20b5a900-f2b5-11ea-8d04-e039a6b38750.png">

Ideally, we should round the numbers to make these match. IIRC the difference comes from the units system calculating with partial values vs the Moveable not (I might be mistaken).

Reply:
That looks like another issue, can you share the story data dump?

Reply:
Of course now when you ask it's not happening anymore. All good then, I suppose.

Reply:
Ok, let's see if Claudio will find it.